### ISSUE TITLE
do we uses plain CSS for the project?

### ISSUE THREAD

**Issue by edgarfb** [twentymurial33/Tip_calculator#1]

**Comment by twentymurial33** [comment]
Yes plan CSS is fine...create your own branch ...don't commit to master

**Comment by edgarfb** [comment]
Ok, I'm new working with others. Please tell me if I'm wrong with anything.
I just created a branch 'edgar' and I will work in that branch and you guys can see it and I will see your branch. I'm ok?
what about the 'master' branch?
